source/client/renderer/Textures.cpp

```diff
@@ -19,9 +19,19 @@ int Textures::loadTexture(const std::string& name, bool bIsRequired)
 
 	Texture t = m_pPlatform->loadTexture(name, bIsRequired);
 
-	int result = -1;
-	if (t.m_pixels)
-		result = assignTexture(name, t);
+	if (!t.m_pixels) {
+		t.field_C = 1;
+		t.field_D = 0;
+		t.m_width = 2;
+		t.m_height = 2;
+		t.m_pixels = new uint32_t[4];
+		t.m_pixels[0] = 0xfff800f8;
+		t.m_pixels[1] = 0xff000000;
+		t.m_pixels[3] = 0xfff800f8;
+		t.m_pixels[2] = 0xff000000;
+	}
+
+	int result = assignTexture(name, t);
```

Comment:
It looks like -1 was originally returned here to indicate an error occurred loading the texture. Surely there's a way to still allow for error detection & "texture not found" warnings while still keeping the placeholder textures.

Reply:
Since a texture is now always assigned, IMO, there's no reason for `Textures` to propagate the error.

Also, warnings still work. They're printed by `AppPlatform`.

Reply:
As long as warnings still work, then I don't really see any downsides here. Looks all good!

Reply:
What duplicate check?

Reply:
Oh! Sorry, I was looking in my mail and saw something that looked like this. Totally didn't see the `-` and `+` signs, sorry
![image](https://github.com/ReMinecraftPE/mcpe/assets/17717494/9ed823cc-4465-4ebf-b11c-b75810ebf4e1)